test/mocha/helpers.js

```diff
@@ -57,10 +57,7 @@ exports.sequenceNumberTests = [
   ['in range [2**32, MAX_SAFE_INTEGER] (middle)',
     2 ** 32 + (Number.MAX_SAFE_INTEGER - 2 ** 32 - 1) / 2],
   ['in range [2**32, MAX_SAFE_INTEGER] (random)',
-    getRandomIntInclusive(2 ** 32, Number.MAX_SAFE_INTEGER)],
-  // near Number.MAX_SAFE_INTEGER
-  ['MAX_SAFE_INTEGER-1', Number.MAX_SAFE_INTEGER - 1],
```

Comment:
Should add back the MAX - 1 test since that's the edge case now.

Reply:
We are using this same export to also test the update which increases the sequence by 1. I changed it to MAX_SAFE_INTEGER - 2 so it will test the ceiling on update.